packages/tools/pie-icons-vue/test/IconAlcohol.test.js

```diff
@@ -29,4 +37,55 @@ describe('IconAlcohol', () => {
         expect(wrapper.classes()).toContain(`c-pieIcon--${iconSize}`);
         expect(wrapper.classes()).toHaveLength(3);
     });
+
+    test.each([
+        'xxs',
+        'xxxl',
+        '',
+    ])('should not allow invalid sizes - %s', (iconSize) => {
+        // Arrange
+        const consoleError = console.error;
+        const errorMock = vi.fn();
+        console.error = errorMock;
+        const propsData = { iconSize };
+
+        // Act
+        const wrapper = mount(IconAlcohol, { propsData });
+
+        // Assert
+        expect(wrapper.classes()).not.toContain(`c-pieIcon--${iconSize}`);
+        expect(wrapper.classes()).toContain('c-pieIcon--xs'); // Default size
+        expect(errorMock.mock.calls[0][0].startsWith('[Vue warn]: Invalid prop')).toBeTruthy();
+
+        console.error = consoleError;
```

Comment:
You should be able to just mock the implementation on line 47 and then restore the mock later (normally you can get away with just mocking it without restoring it).